static/app/components/feedback/widget/useFeedbackWidget.tsx

```diff
@@ -32,15 +33,17 @@ export default function useFeedbackWidget({buttonRef, messagePlaceholder}: Props
 
     if (buttonRef) {
       if (buttonRef.current) {
-        const widget = feedback.attachTo(buttonRef.current, options);
-        return () => {
-          feedback.removeWidget(widget);
-        };
+        // TODO: Remove this after we update to 8.0.0-beta.5
+        // @ts-expect-error This is wrongly typed in the current version...
+        return feedback.attachTo(buttonRef.current, options);
       }
     } else {
-      const widget = feedback.createWidget(options);
-      return () => {
-        feedback.removeWidget(widget);
+      // TODO: Remove this after we update to 8.0.0-beta.5
+      // @ts-expect-error This is wrongly typed in the current version...
+      const widgetPromise = feedback.createWidget(options);
+      return async () => {
+        const widget = await widgetPromise;
+        widget.removeFromDom();
       };
```

Comment:
This is broken in the current SDK, needs https://github.com/getsentry/sentry-javascript/pull/11838